src/server/dflycmd.h

```diff
@@ -91,8 +92,9 @@ struct FlowInfo {
 //
 class DflyCmd {
  public:
-  // See header comments for state descriptions.
+  // See class comments for state descriptions.
   enum class SyncState { PREPARATION, FULL_SYNC, STABLE_SYNC, CANCELLED };
+  static std::string_view SyncStateName(DflyCmd::SyncState sync_state);
```

Comment:
I dont see we are calling this function from outside dflycmd.cc so we can move it from the header and move the impl to the unnamed namespace